app/pages/sections/order-items/catalogue-solutions/delete/confirmation/controller.js

```diff
@@ -0,0 +1,13 @@
+import { getContext } from './contextCreator';
+
+export const getDeleteCatalogueSolutionConfirmationContext = async ({
+  req,
+  sessionManager,
+  accessToken,
+  logger,
+}) => {
+  const { orderId } = req.params;
+  const { solutionName } = req.params;
+
+  return getContext({ orderId, solutionName });
+};
```

Comment:
There are some linting issues in this file:

```shell
app/pages/sections/order-items/catalogue-solutions/delete/confirmation/controller.js
  5:3  error  'sessionManager' is defined but never used  no-unused-vars
  6:3  error  'accessToken' is defined but never used     no-unused-vars
  7:3  error  'logger' is defined but never used          no-unused-vars
```

Reply:
resolved